.pipelines/stages/jobs/py-linux-cpu-packaging-job.yml

```diff
@@ -0,0 +1,136 @@
+parameters:
+  - name: arch
+    type: string
+    default: 'x64'
+jobs:
+- job: Linux_CPU_Wheels
+  strategy:
+    matrix:
+      x64:
+        buildArch: 'x64'
+        machine_pool: 'onnxruntime-Ubuntu2204-AMD-CPU'
+        docker_arch: 'x64'
+      ${{ if eq(parameters.arch, 'arm64') }}:
+        arm64:
+          buildArch: 'arm64'
+          machine_pool: 'onnxruntime-linux-ARM64-CPU-2019'
+          docker_arch: 'aarch64'
+  timeoutInMinutes: 240
+  workspace:
+    clean: all
+  pool: $(machine_pool)
+  variables:
+  # The build machine pool doesn't have dotnet, so it can't run CG.
+  - name: skipComponentGovernanceDetection
+    value: true
+  - name: ort_version
+    value: '1.17.0'
+
+  steps:
+  - checkout: self # due to checkout multiple repos, the root directory is $(Build.SourcesDirectory)/onnxruntime-genai,
+    clean: true
+    submodules: recursive
+
+  - checkout: manylinux # due to checkout multiple repos, the root directory is $(Build.SourcesDirectory)/manylinux,
+    submodules: false
+
+  - template: steps/set-nightly-build-option-variable-step.yml
+
+  - task: DownloadGitHubRelease@0
+    inputs:
+      connection: 'GitHub - Release'
+      userRepository: 'microsoft/onnxruntime'
+      defaultVersionType: 'specificTag'
+      version: 'v$(ort_version)'
+      itemPattern: 'onnxruntime-linux-$(buildArch)-$(ort_version)*'
+      downloadPath: '$(Build.SourcesDirectory)'
+    displayName: Download ONNXRuntime
+      -
+  - task: ExtractFiles@1
+    inputs:
+      archiveFilePatterns: '**/*.tgz'
+      destinationFolder: '$(Build.SourcesDirectory)/onnxruntime-genai'
+      cleanDestinationFolder: false
+      overwriteExistingFiles: true
+    displayName: Unzip OnnxRuntime
+
+  - bash: |
+      mv  onnxruntime-linux-$(buildArch)-$(ort_version) ort
+    displayName: Rename Onnxruntime to ort
+    workingDirectory: '$(Build.SourcesDirectory)/onnxruntime-genai'
+
+  - script: |
+      set -e -x
+      az login --identity -u /subscriptions/00c06639-6ee4-454e-8058-8d8b1703bd87/resourcegroups/proj-onnxruntime-dev/providers/Microsoft.ManagedIdentity/userAssignedIdentities/ortcibuild_readonly
```

Comment:
```
az login --identity -u 63b63039-6328-442f-954b-5a64d124e5b4
```

Use an ID is better. Because an ID doesn't get changed if the resource is moved to another resource group.

Reply:
Tried but it didnt work
https://aiinfra.visualstudio.com/ONNX%20Runtime/_build/results?buildId=418186&view=logs&j=e8617949-7cc5-55ad-1f2e-d6930d975d4f&t=1fd12ec2-da31-59e4-39ed-d2b202af722b&l=12

Reply:
Because you missed one digit when you copy-paste!